pkg/kf/commands/apps/apps.go

```diff
@@ -54,13 +54,16 @@ func NewAppsCommand(p *config.KfParams, appsClient apps.Client) *cobra.Command {
 				for _, app := range applist {
 
 					// Requested State
-					requestedState := "started"
-					if app.Spec.Instances.Stopped {
-						requestedState = "stopped"
-					} else if cond := app.Status.GetCondition("Ready"); cond != nil && cond.Status == "Pending" {
-						requestedState = "starting"
-					} else if !app.DeletionTimestamp.IsZero() {
+					var requestedState string
+					switch {
+					case !app.DeletionTimestamp.IsZero():
 						requestedState = "deleting"
+					case app.Spec.Instances.Stopped:
+						requestedState = "stopped"
+					case !app.Status.IsReady():
```

Comment:
this should be higher in the switch statement, probably right below deletion checks because it encompasses much more than just whether an app is started, or stopped. Otherwise, the code could get the single condition associated with the status of the KService and map it here while breaking out the overall status in another column.

Reply:
maybe that can be another PR

Reply:
I think I would expect that information to be in kf app instead. WDYT?

Reply:
I think I'd want to see the actual status in this table because kf's state machine is much larger than cf's. I don't want users going down the wrong rabbit hole if `requestedState` is out of sync with reality or something that didn't used to be part of the app lifecycle (like bindings or routes) is now affecting the requested state

Reply:
Hmm. I think I would want to see a mock up of what you're thinking. I'm having a hard time imagining how we would concisely show that information.

With that in mind, I agree it might be best to do in a different PR.